include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -50,6 +50,15 @@ POSSIBILITY OF SUCH DAMAGE.
 namespace libtorrent {
 namespace dht
 {
+	// This structure hold the relevant counters for the storage
+	struct TORRENT_EXPORT dht_storage_counters
+	{
+		boost::int64_t torrents;
```

Comment:
do you think 32 bit ints are too narrow here? I realize you presumably made them 64 bits just like the counters class, but unless there's doubt that int32_t is enough, I think it would be better to go with that